tests/h/views/activity_test.py

```diff
@@ -8,7 +8,7 @@
 
 from h.views import activity
 
-@pytest.mark.usefixtures('paginate', 'query')
+@pytest.mark.usefixtures('paginate', 'query', 'pyramid_request_mocked')
```

Comment:
I am not familiar at all with what you're suggesting or really what the code is doing. renaming and removing the useFixture reference did not override it. I'm afraid for this suggestion, if it's worth holding up the merge, you may need to do the code change yourself so I can understand/learn what you are talking about